tests/release/e2e-test-default-with-deployment.go

```diff
@@ -210,26 +210,38 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1199]test-release-e2e-with-deploy
 			}, releasePipelineRunCompletionTimeout, defaultInterval).Should(BeTrue())
 		})
 
-		It("tests a Release should have been created in the dev namespace and succeeded.", func() {
+		It("tests a Release exists", func() {
 			Eventually(func() bool {
 				releaseCreated, err := fw.AsKubeAdmin.ReleaseController.GetFirstReleaseInNamespace(devNamespace)
 				if releaseCreated == nil || err != nil {
 					return false
 				}
 
-				return releaseCreated.IsReleased()
+				return true
 			}, releaseCreationTimeout, defaultInterval).Should(BeTrue())
 		})
-	})
 
-	It("tests a Release should report the deployment was successful.", func() {
-		Eventually(func() bool {
-			releaseCreated, err := fw.AsKubeAdmin.ReleaseController.GetFirstReleaseInNamespace(devNamespace)
-			if releaseCreated == nil || err != nil || len(releaseCreated.Status.Conditions) < 2 {
-				return false
-			}
+		// phase Deployed happens before phase Released
+		It("tests a Release exists and was deployed successfully", func() {
+			Eventually(func() bool {
```

Comment:
I think it'd be better if functions passed to `Eventually` would have `error` type return value to prevent "swallowing" errors, but given the fact that this will be generally addressed in [other PR](https://github.com/redhat-appstudio/e2e-tests/pull/558), I'm happy with merging this one

Reply:
Thanks, @psturc!